services/scheduler/src/main/java/org/mule/service/scheduler/internal/DefaultSchedulerService.java

```diff
@@ -282,15 +286,15 @@ public void start() throws MuleException {
 
     cpuLightExecutor = new ThreadPoolExecutor(threadPoolsConfig.getCpuLightPoolSize(), threadPoolsConfig.getCpuLightPoolSize(),
                                               0, SECONDS, new LinkedBlockingQueue<>(threadPoolsConfig.getCpuLightQueueSize()),
-                                              new SchedulerThreadFactory(cpuLightGroup));
+                                              new SchedulerThreadFactory(cpuLightGroup), byCallerThreadGroupPolicy);
```

Comment:
The AbortPolicy is not made configurable as part of this PR from what I can see. That might not be needed, but it doesn't match the JIRA issue.